cluster/plan.go

```diff
@@ -265,6 +267,11 @@ func (c *Cluster) BuildKubeAPIProcess(host *hosts.Host, serviceOptions v3.Kubern
 	}
 	if c.Services.KubeAPI.AuditLog != nil && c.Services.KubeAPI.AuditLog.Enabled {
 		Binds = append(Binds, fmt.Sprintf("%s:/var/log/kube-audit:z", path.Join(host.PrefixPath, "/var/log/kube-audit")))
+		bytes, _ := yaml.Marshal(c.Services.KubeAPI.AuditLog.Configuration.Policy)
```

Comment:
Can we print or log the error in case of the malform input?

Reply:
Added, thanks